### PR TITLE
feat: #785 add menor flag

### PR DESCRIPTION
**Pull request by AKiroi**
- added mentor flag 
image

### CODE REVIEW

File: mw-webapp/src/component/userCard/UserCard.module.scss [hunk 1]
```diff
@@ -83,3 +85,11 @@ $userCardAdditionalHeight: 82px;
 .ways {
   flex-wrap: wrap;
 }
+
+.flagMentor {
+  position: absolute;
+  top: 0;
+  right: $flagMenorRightOffset;
+  fill: var(--bookmarksIconColor);
+  stroke: none
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/userCard/UserCard.module.scss:94]:
```suggestion
  stroke: none;
```

Comment (Ekaterina1994) [inline mw-webapp/src/component/userCard/UserCard.module.scss:94]: Why stroke is 0?

Comment (AKiroi) [reply mw-webapp/src/component/userCard/UserCard.module.scss:94]: in figma maket we don't have stroke, only fill 
image

Comment (AKiroi) [reply mw-webapp/src/component/userCard/UserCard.module.scss:94]: done

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**